plugin/pkg/admission/limitranger/admission.go

```diff
@@ -421,10 +421,7 @@ func (d *DefaultLimitRangerActions) SupportsAttributes(a admission.Attributes) b
 		return false
 	}
 
-	// Since containers and initContainers cannot currently be added, removed, or updated, it is unnecessary
-	// to mutate and validate limitrange on pod updates. Trying to mutate containers or initContainers on a pod
-	// update request will always fail pod validation because those fields are immutable once the object is created.
-	if a.GetKind().GroupKind() == api.Kind("Pod") && a.GetOperation() == admission.Update {
+	if a.GetKind().GroupKind() == api.Kind("Pod") && a.GetOperation() == admission.Update && !feature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) {
```

Comment:
FYI, This won't work after #128266 as we add `resize` subresource to do pod resize.

Reply:
Do we need  this logic   after using resize subresource ? @AnishShah

Reply:
Yes, we still need the logic. But instead of pod update, we need it during resize subresource. I provisionally added the logic in my PR

Reply:
so this pr can close after your pr merged ? and e2e test can be writen in your pr ? @AnishShah

Reply:
Yes, I added e2e tests in  https://github.com/kubernetes/kubernetes/pull/128266/commits/1b93d0f989ab8bc43d3e3ac84ad5ed3bac08d0c0